src/js/environment/environment.ts

```diff
@@ -45,12 +44,21 @@ function supportsPassive(): boolean {
     return passiveOptionRead;
 }
 
+const isWindows = () => {
+    return userAgent.indexOf('Windows') > -1;
+};
```

Comment:
Since this is only used in one place, move the implementation into `get windows()`.

Really this whole module was supposed to replace `browserVersion` and `osVersion` but instead was added as a layer on top. Then this function was declared up here as something missing from the tools this module was supposed to deprecate. Treat it as such.